usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -338,16 +339,25 @@ def insert_new_fpds(self, to_insert, total_rows):
 
     def perform_load(self, ids_to_delete, ids_to_insert):
 
-        if len(ids_to_delete) > 0:
-            with timer("deletion of all stale FPDS data", logger.info):
-                self.delete_stale_fpds(ids_to_delete=ids_to_delete)
-        else:
+        delete_index = 0
+        while delete_index < ids_to_delete.__len__():
```

Comment:
In Python it is `len(<list/dict/...>)` same for line 345, 354, 357.

If a dunder is accessed directly there is probably a better way.